policies/validation.rb

```diff
@@ -0,0 +1,8 @@
+module Validation
+  def self.dashboard_params_invalid?(params)
+    params[:departure_id].nil? ||
+    params[:departure_id].empty? ||
+    params[:arrival_id].nil? ||
+    params[:arrival_id].empty?
```

Comment:
Use class instead of module
Also, extract `nil` & `empty` logic to separate method so that you have something like
`blank_value?(params[:departure_id]) || blank_value?(params[:arrival_id])`